homeassistant/components/rfxtrx/binary_sensor.py

```diff
@@ -206,6 +216,15 @@ def unique_id(self):
         """Return unique identifier of remote device."""
         return self._unique_id
 
+    @property
+    def device_info(self):
+        """Return the device info."""
+        return {
+            "identifiers": {(DOMAIN, *self._device_id)},
```

Comment:
Why not use `id_string` ?

Reply:
id_string is not really unique from what i can tell. it can overlap between packet types.

Reply:
and, i'm resolving some things related to that in the drop event pull.